src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Repositories/Catalog/DemoCatalogRepository.cs

```diff
@@ -17,13 +19,24 @@ public DemoCatalogRepository(DemoCatalogDbContext dbContext)
         {
         }
 
+        public override async Task<ItemEntity[]> GetItemByIdsAsync(string[] itemIds, string responseGroup = null)
+        {
+            var result = await base.GetItemByIdsAsync(itemIds, responseGroup);
+            var itemResponseGroup = EnumUtility.SafeParseFlags(responseGroup, ItemResponseGroup.ItemLarge);
+            if (!itemIds.IsNullOrEmpty() && result.Any() && itemResponseGroup.HasFlag(ItemResponseGroup.ItemLarge))
```

Comment:
Add empty string before and after if statement

Reply:
Fixed